File: Engr-T.stark/reverseBits.cpp
class Solution {
public:
    uint32_t reverseBits(uint32_t n) {
        int result=0;
        int num = 0;
        while(num<32) {
            result=result<<1;
            if(n&1) {
                result=result|1;
            }
            n=n>>1;
            num++;
        }
        return result;
    }
};

File: Josephineoe/day_15/reverseBits.py
class Solution:
    def reverseBits(self, n: int) -> int:
        reverse = 0
    #day 14 josephineoe
        for i in range(32):
           # remainder = n % 10 
            remainder = (n >> i) & 1
            reverse = reverse | (remainder << (31 - i)) #reverse*10 + remainder
            
        return reverse
    # n == 123456..didnt work
    # print(str(n)[::-1]) didnt work...